flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1201,13 +1201,19 @@ bool validInputRange(int16_t min, int16_t max, uint16_t value, uint16_t offset)
  */
 static void applyDeadband(float *value, float deadband)
 {
-	if (fabsf(*value) < deadband)
+	if (deadband < 0.00001f) return;
```

Comment:
I think what you're doing here is choosing the smallest number such that we still have precision over a 32-bit float. Is that correct? If so, could the magic number become a ```#define``` instead?